core/package.json

```diff
@@ -62,7 +62,7 @@
     "build.vendor": "rollup --config ./scripts/swiper.rollup.config.js",
     "build.css": "npm run css.sass && npm run css.minify",
     "build.debug": "npm run clean && stencil build --debug",
-    "build.docs.json": "stencil build --docs-json dist/docs.json",
+    "build.docs.json": "stencil build --docs-json ../docs/core.json",
```

Comment:
I would remove this, and add a new outputTarget in stencil.config.ts, so we update the .json in every build, just like markdown (it will be also much faster, since we reuse the same data)

Reply:
Good call. I just updated the PR to use an outputTarget.